vcf/queryInfo.go

```diff
@@ -109,28 +109,30 @@ func parseValue(v Vcf, s []string, k Key) interface{} {
 
 		for i := range data {
 			if !checkNumber(v, k, len(data[i])) {
-				log.Panicf("unexpected number of values")
+				log.Panicf("unexpected Number of values")
 			}
 		}
 
 		return data
 
-	case typeString:
+	case String:
 		data := make([][]string, len(s))
 		for i := range s {
-			stringValues = strings.Split(s[i], ",")
-			data[i] = stringValues
+			//stringValues = strings.Split(s[i], ",")
+			//data[i] = stringValues
+			data[i] = []string{s[i]} // many fields ignore specifications and use literal commas
 		}
 
-		for i := range data {
-			if !checkNumber(v, k, len(data[i])) {
-				log.Panicf("unexpected number of values")
-			}
-		}
+		// Illegal string fields are common, so we will suppress this check.
```

Comment:
Is there a common thing that happens we could allow, but still have some checks?  What kind of stuff commonly happens in files?  Does this make it hard to apply the filter criteria that I user may specify?